README.md

```diff
@@ -27,13 +27,13 @@ Commit your code regularly and meaningfully. This helps both you (in case you ev
 Be prepared to demonstrate your understanding of this week's concepts by answering questions on the following topics. You might prepare by writing down your own answers before hand.
 
 1. What is the DOM?
-
+the document object model is where the base model of a document that can be manipulated
```

Comment:
the DOM is an object representation of your HTML that can be manipulated via Javascript.